src/pseudopeople/noise_functions.py

```diff
@@ -595,7 +536,7 @@ def use_fake_names(
     :param column_name: String for column that will be noised, will be the key for RandomnessStream
     :return:
     """
-    column = data[column_name]
+
     fake_first = fake_first_names
     fake_last = fake_last_names
```

Comment:
Why are we doing these assignments? It seems like we should just be able to use `fake_first_names` and `fake_last_names` directly.

Reply:
Ill see if I can remove this.